scripts/get-user.py

```diff
@@ -17,8 +17,6 @@
     ./get-user.py --api-url=localhost:5000 --api-token=myToken supplier dos2 digital-outcomes
 """
 
-from __future__ import print_function
```

Comment:
The script will definitely break on Py2 without it, but since it's only used by developers locally that's probably fine at this point.

Reply:
Oh yeah. I forgot it doesn't mind the brackets on their own.